src/cli/cmd/login/mod.rs

```diff
@@ -40,8 +49,61 @@ impl CommandExecute for LoginSubcommand {
     }
 }
 
+pub async fn dnee_uds() -> color_eyre::Result<SendRequest<axum::body::Body>> {
+    let dnee_uds_socket_dir = Path::new(&DETERMINATE_STATE_DIR);
+    let dnee_uds_socket_path: PathBuf = dnee_uds_socket_dir.join(DETERMINATE_NIXD_SOCKET_NAME);
+
+    let stream = TokioIo::new(UnixStream::connect(dnee_uds_socket_path).await.unwrap());
+    let (mut sender, conn): (SendRequest<Body>, _) =
+        hyper::client::conn::http1::handshake(stream).await.unwrap();
+
+    // NOTE(colemickens): for now we just drop the joinhandle and let it keep running
+    let _join_handle = tokio::task::spawn(async move {
+        if let Err(err) = conn.await {
+            println!("Connection failed: {:?}", err);
```

Comment:
We'll want to use tracing instead of println

Reply:
done